src/pt/mangaonline/build.gradle

```
@@ -0,0 +1,10 @@
+ext {
+    extName = 'Manga Online'
+    extClass = '.MangaOnline'
+    themePkg = 'madara'
+    baseUrl = 'https://mangaonline.app'
+    overrideVersionCode = 1
```

the initial version when themepkg is used is 0
```suggestion
    overrideVersionCode = 0
```